### PR TITLE
Added tests for enhanced table head component

### PR DESCRIPTION
**Pull request by MickhaelSh**
Added tests for enhanced table head component

### CODE REVIEW

File: tests/unit/components/enhanced-table/enhanced-table-head/EnhancedTableHead.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            onSelectAllClick={ onSelectAllClickMock }
+          />
+        </table>
+      </TableProvider>
+    )
+  })
+
+  it('renders all columns in the table', () => {
+    const nameColumn = screen.getByText(columnsMock[0].label)
+    const emailColumn = screen.getByText(columnsMock[1].label)
```
Comment (Tolik170) [inline tests/unit/components/enhanced-table/enhanced-table-head/EnhancedTableHead.spec.jsx:38]: Is it necessary to check name and emai? maybe one of them would be enough

File: tests/unit/components/enhanced-table/enhanced-table-head/EnhancedTableHead.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    
+    expect(nameColumn).toBeInTheDocument()
+    expect(emailColumn).toBeInTheDocument()
+  })
+  it('Should show table actions when isSelection is true', () => {
+    const actionColumn = screen.getByText('table.actions')
```
Comment (Tolik170) [inline tests/unit/components/enhanced-table/enhanced-table-head/EnhancedTableHead.spec.jsx:44]: As i see, tableActions or smth like that would be more appropriate

File: tests/unit/components/enhanced-table/enhanced-table-head/EnhancedTableHead.spec.jsx (hunk 1 above)
Comment (MickhaelSh) [reply tests/unit/components/enhanced-table/enhanced-table-head/EnhancedTableHead.spec.jsx:38]: no, it is not necessary, but I have two columns in mock. decided to check both

File: tests/unit/components/enhanced-table/enhanced-table-head/EnhancedTableHead.spec.jsx (hunk 2 above)
Comment (MickhaelSh) [reply tests/unit/components/enhanced-table/enhanced-table-head/EnhancedTableHead.spec.jsx]: agree, fixed

**Review by Tolik170 — APPROVED** [review]

**Review by KhrystynaPavlikovska — APPROVED** [review]

**Review by Yurenko — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**